### ISSUE TITLE
Clicking on a future date doesn't create a journal entry correctly

### ISSUE THREAD

**Issue by jonahgoldsaito** [xyhp915/logseq-journals-calendar#5]
First off, awesome. Thanks for making this plugin!

Yesterday I wanted to create a journal entry for today, i.e. one day in advance. An attempt at planning / structuring my day. 
Today the app opened with an empty journal page for the day, and today's journal page was empty.

```
// This is page data for the page I made in advance via the journal calendar plugin simply by clicking on a date in the future and editing the resulting page:

{:db/id 252,
 :block/created-at 1635216741402,
 :block/file {:db/id 254},
 :block/name "2021-10-26",
 :block/original-name "2021-10-26",
 :block/unordered true,
 :block/updated-at 1635253224523,
 :block/uuid #uuid "61776d63-0a91-451a-a185-4ea1471429a0"}
```

```
// The data for today's empty auto-created journal page when I launched the app
// Clearly a different page, but also a very different format

{:db/id 259,
 :block/format :markdown,
 :block/journal-day 20211026,
 :block/journal? true,
 :block/name "oct 26th, 2021",
 :block/original-name "Oct 26th, 2021",
 :block/uuid #uuid "61777d42-13a9-406c-840e-89dfc49aba34"}

```

```
// When I try creating a test of a page for tomorrow via logseq's /date date picker component, the format appears to be the same as the auto-generated. I assume it will work as intended.

{:db/id 268,
 :block/journal-day 20211027,
 :block/journal? true,
 :block/name "oct 27th, 2021",
 :block/original-name "Oct 27th, 2021",
 :block/uuid #uuid "6177ff89-c64d-4ac3-af6f-b0f37c72f12c"}
```

Is there a reason the calendar plugin doesn't use a different format? Would love it to work as expected :)

**Comment by MelBourbon** [comment]
With current Logseq 0.7.1 using the plugin also creates the date file not in the journal folder. It creates instead a file in the pages folder. With this the new created files are not part of the journal.

**Comment by kweiberth** [comment]
Was this a conscious decision that this plugin would no longer create journal entries? Or was it a breaking change in Logseq, and this plugin has yet to be updated?

**Comment by Robinlovelace** [comment]
Woa this sounds like a major bug. I'm thinking about installing the plugin but not sure it's ready after reading this. Did you get it working on recent versions @jonahgoldsaito and please give us an update @xyhp915 if you get a chance.

Many thanks for an amazing plugin, look forward to testing it!

**Comment by xyhp915** [comment]
I tested with the latest version (0.7.8). And it works fine?

https://user-images.githubusercontent.com/1779837/180190318-b1fc42df-c9ef-4504-b77c-9391e2dcdf7a.mp4

**Comment by kweiberth** [comment]
Works for me now as well w/ version 0.7.8.

**Comment by MelBourbon** [comment]
Still not working for me in 0.7.8 ... Maybe related to date format used. If I create a page with the plugin for next Tuesday for example it creates the file with `Tu, 26-07-2022`. 
![image](https://user-images.githubusercontent.com/30600002/180210297-1cfd5853-d0bf-45ae-978c-b946024b4e7d.png)

But my time format is set to:

![image](https://user-images.githubusercontent.com/30600002/180210381-caf9f14f-725e-4e36-8134-0ed0c6da757b.png)

And there normally `Tue, 27-07-2022` is required. Therefore this is a normal page only and not stored in journals folder

**Comment by JRSerjeant** [comment]
> stalling the plugin but not sure it's ready after reading this. D

I have this same issue. Does this problem present if you have a custom date format in  `config.edn` > `:journal/page-title-format "E dd MMMM yyyy"`

Not had a chance to test this yet/

**Comment by artjomsR** [comment]
I can create an entry via the calendar, but the same calendar won't show that a day has any entries there. Refreshing and re-indexing doesn't help - only after logseq restart the day is correctly populated in the calendar plugin. See video demonstration

# My system specs
- Win 10
- logseq 0.10.6
- Journals calendarv0.10.10

https://github.com/xyhp915/logseq-journals-calendar/assets/6875790/17fba698-6b3d-4ade-99f2-ed1b27d2d5d8

![image](https://github.com/xyhp915/logseq-journals-calendar/assets/6875790/85b192ee-2f1e-467c-a498-9d2b908515cc)
